### ISSUE TITLE
bash: dot: command not found

### ISSUE THREAD

**Issue by seongeunC** [ConsenSys/surya#81]
In OSX

$ surya graph contracts/MyContract.sol | dot -Tpng > MyContract.png
bash: dot: command not found

**Comment by seongeunC** [comment]
i solved it
$ brew install graphviz

**Comment by onggunhao** [comment]
We should put a reference to this in the README

**Comment by maurelian** [comment]
Good point, I did that.

**Comment by GNSPS** [comment]
We can later merge to `master` but previous convention would say commits should go to `develop` instead. 😊

**Comment by maurelian** [comment]
Ah, sorry.

**Comment by AndreiD** [comment]
please update the fucking readme. thanks

**Comment by GNSPS** [comment]
Can you tell me what you'd like to see updated in the fucking readme, @AndreiD, please? 😊 🙏

**Comment by AndreiD** [comment]
$ brew install graphviz or the equivalent in linux. obviously

**Comment by GNSPS** [comment]
If it is "obvious" I wonder why the note already in the README:

<img width="838" alt="Captura de ecrã 2021-11-19, às 18 02 31" src="https://user-images.githubusercontent.com/4008213/142670196-24a2f3e0-effa-4aa4-a942-c47329f87a5a.png">

isn't enough.

Specifically, your request about adding the Homebrew command, presupposes I'd have to exhaustively enumerate the commands for package managers on all the OSes which is impractical. 😄 

So, I think for now I will leave the fucking README as is and leave the command of the package manager of choice as an exercise to the reader.

Thank you for the suggestion, though. 😊 🙏

**Comment by AndreiD** [comment]
there are 8 upvotes in the second reply here. Obviously people fail at this. Your job is to make it easier ...otherwise why you bother with the readme at all? why can't just we fucking read the source code and figure out what this is.
Consensys pays you for this, do a good job please :)

**Comment by GNSPS** [comment]
Let me leave you a piece of advice for the rest of your life.

Please do not mistreat people that are building stuff for the community for free. It is because of people like you that OSS is not more widespread.

Also that bitterness will only do you a disservice in the long run. It is a much smarter long-term play to be kind. Even though the results might be delayed in comparison to being mean and caustic, they are much more long-lasting.

As an illustration of my point above, I will, from now on, always try to keep some distancing from you and any business I am involved with. This obviously extends to anyone that asks me for any kind of input about you.
The information will not travel fast, might not even impact you at all, but what you can be entirely sure of is that it won't change.

It's a closed door for life. 🤷‍♂️ 😞

**Comment by AndreiD** [comment]
you ain't doing any OSS mate, you're being paid to do this so consensys have something pretty for their portfolio. As for the life lesson, it was so boring I didn't even read it. Can you just do a TLDR please

**Comment by AndreiD** [comment]
ah I get it, when someone asks me about me you'll say I'm a jerk :)) ok mate. but for Christ or Buddha or whatever god you believe in, ### **can you please update the readme file with the install of the "dot" command ?** :)))
